web/mass_class/rtmp_bridge.py

```
@@ -0,0 +1,178 @@
+        # In a real implementation, this would update the WebRTC connection
+        pass
+
+    async def stop_stream(self, event):
+        session_id = event["session_id"]
+
+        if session_id in self.processes:
+            process = self.processes[session_id]
+            try:
+                process.terminate()
+                await process.wait()
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)